app/javascript/react/components/animals/AnimalIndexTile.js

```diff
@@ -2,10 +2,15 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 
 const AnimalIndexTile = props => {
+  let buttons
+  if(props.showButton){
+    buttons = <div><button>Edit</button><br/><button>Delete</button></div>
```

Comment:
this should be indented, not all one line. Or better yet, define a really small component for it